consensus/pow/worker.go

```diff
@@ -0,0 +1,10 @@
+package consensus
+
+// Worker is an PoW engine.
+type Worker interface {
+    // Returns the current mining result of a PoW consensus engine.
+    Producer() int
```

Comment:
Producer() int
better to return a string if you want it to be general

Reply:
nice suggestion. fixed.